components/ui/CompactFeature.vue

```diff
@@ -3,12 +3,12 @@
     <img
       v-if="icon"
       class="compact-feature__icon"
-      :src="`/images/icons/icon${icon}White@3x.png`"
+      :src="`/images/icons/${icon}.svg`"
     >
-    <h2 v-if="title" class="compact-feature__title">
+    <h2 v-if="title" class="compact-feature__title copy__subtitle">
```

Comment:
As I see it, `h2` should be used for the section's titles. I think we should use here an `h3`, the same way we did [for cards](https://github.com/Qiskit/qiskit.org/blob/e3ffdf1ab5343fb08db753e91c3483283b4042bf/components/ui/AppCard.vue#L9)